### DISCUSSION TITLE
getRadius in ColumnLayer ?

### DISCUSSION THREAD

**Discussion by mikeldn** [visgl/deck.gl#4324] [Q&A]
Simple question... 
Is there currently, or likely to be, a getRadius method in ColumnLayer?

The purpose of which would be to scale the radius based on a point's data.

ScatterPlotLayer has this 'getRadius' function (inc radiusScale etc).. but I can see how for efficiency you have left it out of ColumnLayer.

Would be great if we could have it though! 

Thanks

**Comment by SymbolixAU** [comment]
Similarly it would be handy for [PointCloudLayer](https://github.com/visgl/deck.gl/blob/master/docs/layers/point-cloud-layer.md) to have a `getRadius` too.

**Comment by Pessimistress** [comment]
You can try [SimpleMeshLayer](https://deck.gl/docs/api-reference/mesh-layers/simple-mesh-layer) with which you can control the scaling in all dimensions

**Comment by charlieforward9** [comment]
Coming soon 👨‍🍳 

https://github.com/visgl/deck.gl/pull/9933

<img width="602" height="788" alt="image" src="https://github.com/user-attachments/assets/9d45e932-7945-4589-842f-5e11e847e8b5" />
